sentry_sdk/hub.py

```diff
@@ -580,25 +535,21 @@ def configure_scope(  # noqa
 
         :returns: If no callback is provided, returns a context manager that returns the scope.
         """
-
-        client, scope = self._stack[-1]
+        scope = Scope.get_isolation_scope()
```

Comment:
Here and in general in the PR it looks like the isolation scope is the "default" scope that we're always returning when we want to do something scope-related. Do we ever use current scopes? What's the usecase for those?

Reply:
The isolation scope is the scope that should be used in the integrations by us. The current scope can be used by the user, but we think it will only be an advanced usage, so the current scope will not be used that often in general.